encord/storage.py

```diff
@@ -996,6 +998,9 @@ def __init__(self, api_client: ApiClient, orm_item: orm_storage.StorageItem):
         self._api_client = api_client
         self._orm_item = orm_item
         self._parsed_metadata: Optional[Dict[str, Any]] = None
+        self._client_metadata_schema: Optional[Dict[str, ClientMetadataSchemaTypes]] = get_client_metadata_schema(
```

Comment:
This should be lazy-loaded; most of the time we don't need it and it's a whole extra network call. Going to be *extremely* slow if you're listing items!

Thinking a bit more of it, I would say "we should have a 'static' (module-level) cache and lazy-load this in `validate_metadata`"; the 'individual item owns a copy of metadata schema" is not scalable in any way.

Reply:
I agree. I think good suggestion. Let me know if what I've implemented looks good now. It's slightly annoying that the schema could change elsewhere and we still would be using the cache